x-pack/plugins/endpoint/common/types.ts

```diff
@@ -311,23 +306,34 @@ export interface LegacyEndpointEvent {
 
 export interface EndpointEvent {
   '@timestamp': number;
+  agent: {
+    id: string;
+    version: string;
+    name: string;
```

Comment:
Do we need `name`? We don't have it in our endpoint schema. Name is used to differentiate between two running agents on the same box: https://github.com/elastic/ecs/blob/master/schemas/agent.yml#L26 I don't think we'll have two on the same box.

Reply:
I don't think so, I'll remove it.